garage/np/algos/batch_polopt.py

```diff
@@ -115,6 +118,155 @@ def log_diagnostics(self, paths):
         self.policy.log_diagnostics(paths)
         self.baseline.log_diagnostics(paths)
 
+    def process_samples(self, itr, paths):
+        baselines = []
+        returns = []
+
+        if hasattr(self.baseline, 'predict_n'):
+            all_path_baselines = self.baseline.predict_n(paths)
+        else:
+            all_path_baselines = [
+                self.baseline.predict(path) for path in paths
+            ]
+
+        for idx, path in enumerate(paths):
+            path_baselines = np.append(all_path_baselines[idx], 0)
+            deltas = path['rewards'] + \
```

Comment:
I think brackets are preferred to break lines here.

Reply:
Thank you for comments! As @naeioi told me this function is outdated and never being used now. So we may update it later completely?

Reply:
which function do you mean, specifically?

Reply:
It is the process_samples() in garage.sampler.BaseSampler in master branch, which I'm trying to move to garage.np.algos.BatchPolopt.